### DISCUSSION TITLE
How to verify the XSS Found result?

### DISCUSSION THREAD

**Discussion by xzhacker123** [mitmproxy/mitmproxy#5990] [Q&A]
I use `mitmproxy -s xss_scanner.py` to scan a site and find that the log prompts "XSS Found" in event log. For example：
```
error: [15:42:16.268] ===== XSS Found ====
error: [15:42:16.268] XSS URL: http://127.0.0.1:9999/vul/xss/test/1029zxcs'd"ao<ac>so[sb]po(pc)se;sl/bsl\eq=3847asd
error: [15:42:16.268] Injection Point: End of URL
error: [15:42:16.268] Suggested Exploit:  onmouseover=alert(0) t=
error: [15:42:16.268] Line: 1029zxcs'd&quot;ao&lt;ac&gt;so[sb]po(pc)se;sl/bsl\eq=3847asd
```
 so how should I verify the scan record? Here is a prompt: "Injection Point: End of URL", I have tried several splicing methods, but nothing works. How should I splice this url so that  Alert popup box can be displayed?
I have tried the following methods to no avail：
```
http://127.0.0.1:9999/vul/xss/test/1029zxcs'd"ao<ac>so[sb]po(pc)se;sl/bsl\eq=3847asd/ onmouseover=alert(0) t=
http://127.0.0.1:9999/vul/xss/test/1029zxcs'd"ao<ac>so[sb]po(pc)se;sl/bsl\eq=3847asd/onmouseover=alert(0) t=
http://127.0.0.1:9999/vul/xss/test/1029zxcs'd"ao<ac>so[sb]po(pc)se;sl/bsl\eq=3847asd onmouseover=alert(0) t=
http://127.0.0.1:9999/vul/xss/test/1029zxcs'd"ao<ac>so[sb]po(pc)se;sl/bsl\eq=3847asdonmouseover=alert(0) t=
http://127.0.0.1:9999/vul/xss/test/ onmouseover=alert(0) t=
http://127.0.0.1:9999/vul/xss/test/onmouseover=alert(0) t=
```

**Comment by Prinzhorn** [comment]
That depends entirely on the injection point and cannot be answered generically. First of all this could be a false positive. Second I'd use dev tools and `view-source` to check where the payload is actually ending up. And then adjust the payload to make it work / valid. And of course you actually need to mouse over the element to trigger the alert with this payload.

**Reply by xzhacker123** [reply]
Tsk for u answer.
